internal/vercheck/vercheck.go

```diff
@@ -20,27 +21,37 @@ import (
 // Than the version in launch.sh, we'll print a warning.
 const expectedLauncherVersion = "v0.1.0"
 
+// currentChannel represents the CLI update channel. For now, we set it to a
+// constant, but we'll expand this to be a variable in the future.
+const currentChannel = "stable"
```

Comment:
The CLI doesn't need to know about channels 

We could, but I would prefer to encapsulate the logic in the launcher only. Having logic in both places introduces coordination problems (as we're currently experiencing)